playwright/fixtures/index.ts

```diff
@@ -35,113 +44,242 @@ export type Fixtures = {
-    const context = await browser.newContext({
-      storageState: {
-        cookies: [],
-        origins: [
-          {
-            origin: config.use.baseURL,
-            localStorage: [
-              {
-                name: "persist:root",
-                value: JSON.stringify({
-                  userContext: {
-                    userReview: {
-                      onboarding: true,
-                      advancedRepository: true,
+      await use({ name: sliceName });
+    },
+
+    /**
+     * Page
+     */
+    page: async ({ browser }, use) => {
+      // Onboard user in Local Storage by default
+      let context = await browser.newContext({
+        storageState: {
+          cookies: [],
+          origins: [
+            {
+              origin: config.use.baseURL,
+              localStorage: [
+                {
+                  name: "persist:root",
+                  value: JSON.stringify({
+                    userContext: {
+                      userReview: {
+                        onboarding: true,
+                        advancedRepository: true,
+                      },
+                      updatesViewed: {
+                        latest: null,
+                        latestNonBreaking: null,
+                      },
+                      hasSeenChangesToolTip: true,
+                      hasSeenSimulatorToolTip: true,
+                      hasSeenTutorialsToolTip: true,
+                      authStatus: "unknown",
+                      lastSyncChange: null,
                     },
-                    updatesViewed: { latest: null, latestNonBreaking: null },
-                    hasSeenChangesToolTip: true,
-                    hasSeenSimulatorToolTip: true,
-                    hasSeenTutorialsToolTip: true,
-                    authStatus: "unknown",
-                    lastSyncChange: null,
-                  },
-                }),
-              },
+                  }),
+                },
+                {
+                  name: "slice-machine_isInAppGuideOpen",
+                  value: "false",
+                },
+              ],
+            },
+          ],
+        },
+      });
+
+      // Prevent user to be onboarded if needed
+      if (!onboarded) {
+        context = await browser.newContext({
+          storageState: {
+            cookies: [],
+            origins: [
               {
-                name: "slice-machine_isInAppGuideOpen",
-                value: "false",
+                origin: config.use.baseURL,
+                localStorage: [],
               },
             ],
           },
-        ],
-      },
-    });
+        });
+      }
+
+      // Create new page object with new context
+      const page = await context.newPage();
+
+      // Logout user by default
+      try {
+        await fs.rm(path.join(os.homedir(), ".prismic"));
+      } catch (error) {
+        // Ignore since it means the user is already logged out
+      }
```

Comment:
Just checking: will this affect the user's actual `.prismic` file? Or is it mocked/virtualized?

Reply:
Yes and it should for me, I saw another comment about this. e2e tests means really e2e as much as possible. Cutting the authentication part is possible but we want to login fully, we will want to fully push, get the envs, changes all. 
We want to be able to see problems that can happen on wroom. If the wroom part change, it should break in our side so we know we have something to do. 

We will differentiate two kind of tests. The standalone one, and the scenario one. At the moment it's only standalone, but later we will add scenarios that will really do everything.